python/paddle/fluid/dygraph/amp/auto_cast.py

```diff
@@ -163,6 +185,7 @@ def check_optimizers(optimizers):
 @signature_safe_contextmanager
 @dygraph_only
 def amp_guard(enable=True,
+              dtype='float16',
```

Comment:
Is it better to make 'dtype' as the last parameter?

Reply:
Done, tks.